test/view/enumerate.cpp

```diff
@@ -0,0 +1,84 @@
+// Range v3 library
+//
+//  Copyright Casey Carter 2018-present
```

Comment:
Why is this copyrighted to me instead of you? I didn't write this test code ;)

Reply:
Right. Copy paste error - my mistake.
Is it ok, if I just put in my github username?

Reply:
Sure.